Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2620,8 +2582,7 @@ def main():
         splunk_parse_raw_command()
         sys.exit(0)
     service = None
-    proxy = params.get('proxy')
-    use_requests_handler = params.get('use_requests_handler')
```

Comment:
The `use_requests_handler` param isn't used anymore?

Reply:
If the user configures proxy, we will use only requests handler (and no longer the deprecated urllib2)